examples/react/src/App.tsx

```diff
@@ -0,0 +1,34 @@
+import hotLogo from './assets/hot_logo.png';
+import { useState } from 'react';
+
+import HotButton from '../../../react/HotButton';
```

Comment:
Perhaps something for the future - but maybe we could make it so this either imports directly from our npm package? - Would look more clear for people looking at the example